__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => {
+	return seed(testData);
+});
+
+afterAll(() => {
+	db.end();
+});
+
+describe("Get", () => {
+	describe("api/categories", () => {
+		test("200-responses with an array of objects", () => {
+			return request(app)
+				.get("/api/categories")
+				.expect(200)
+				.then(({body}) => {
+					expect(body['categories'].length).toBe(4);
+					body['categories'].forEach((categories) => {
+						expect(categories).toHaveProperty("slug");
+						expect(categories).toHaveProperty("description");
```

Comment:
This test is good so far, but you should test for the data type as well.
Consider using something like a toMatchObject, which will allow you to test for data types at the same time as properties